### ISSUE TITLE
Find out where .onion connection attempts are being made

### ISSUE THREAD

**Issue by defuse** [zcash/zcash#651]
I noticed this in my log:

```
2016-01-21 23:49:25 Cannot connect to ubqj4rsu3nqtxmtp.onion:18333: unsupported network
2016-01-21 23:49:26 Cannot connect to i6y6ivorwakd7nw3.onion:18333: unsupported network
2016-01-21 23:49:27 Cannot connect to 4zhkir2ofl7orfom.onion:18333: unsupported network
2016-01-21 23:49:27 Cannot connect to it2pj4f7657g3rhi.onion:18333: unsupported network
2016-01-21 23:49:28 Cannot connect to nkf5e6b7pl4jfd4a.onion:18333: unsupported network
2016-01-21 23:49:29 Cannot connect to t6xj6wilh4ytvcs7.onion:18333: unsupported network
```

**Comment by petjal** [comment]
This file
>https://github.com/Electric-Coin-Company/zcash/blob/293e4f015b811a03e4059cab38c92e7b42978bcc/contrib/seeds/generate-seeds.py
contains
>Script to generate list of seed nodes for chainparams.cpp.
>This script expects two text files in the directory that is passed as an
>argument:
>    nodes_main.txt
>    nodes_test.txt
This file
>https://github.com/Electric-Coin-Company/zcash/blob/293e4f015b811a03e4059cab38c92e7b42978bcc/contrib/seeds/nodes_test.txt
contains
># List of fixed seed nodes for testnet
>
># Onion nodes
>hfsmmn2jbitcoin.onion
>it2pj4f7657g3rhi.onion
>nkf5e6b7pl4jfd4a.onion
>4zhkir2ofl7orfom.onion
>t6xj6wilh4ytvcs7.onion
>i6y6ivorwakd7nw3.onion
>ubqj4rsu3nqtxmtp.onion

**Comment by daira** [comment]
Is this completely fixed by #972?

**Comment by ebfull** [comment]
Yep!
